plugins/plugins.go

```diff
@@ -743,11 +749,16 @@ func (m *Manager) Services() []string {
 	return s
 }
 
-// Logger gets the logger implementation associated with this plugin manager
-func (m *Manager) Logger() sdk.Logger {
+// Logger gets the debug logger for this plugin manager.
```

Comment:
> Adding "default" or "standard" doesn't really clarify the intent. 

Default or standard clarifies intent in that it's the logger people know and recognize from OPA, not some special logger for SDK debugging (though it can certainly be used for that).

> Also, the logging APIs are leveled by design, so we wouldn't have a "debug level logger"...

"debug level logger" == a logger set to debug level

```go
logger := NewStandardLogger()
logger.SetLevel(Debug)
```

Reply:
Did the s/debug/standard/ replacement!